graphics/text/create_caption.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python
+# coding=utf-8
+from PIL import Image, ImageFont, ImageDraw
+import sys
+import os
+
+#create_caption.py <background_image> <title?> <phase?> <outfile>
+
+#try:
+#    font = ImageFont.truetype("DejaVuSansCondensed-Bold.ttf", 11)
+#except IOError:
+font = ImageFont.load_default()
+
+
+txt1= u"© 2001-2017"
```

Comment:
instead of the clutter here, add a `from __future__ import unicode_literals` for Python2 (a no-op in Python3)